client/src/screens/groups.screen.js

```diff
@@ -170,8 +187,8 @@ class Groups extends Component {
     if (user && !user.groups.length) {
       return (
         <View style={styles.container}>
-          <Header onPress={this.goToNewGroup} />
-          <Text style={styles.warning}>{'You do not have any groups.'}</Text>
+          <Header onPress={this.goToNewGroup} onPressFind={this.goToSearchGroup}/>
+          <Text onPress={this.onRefresh} style={styles.warning}>{'You are not a member of any groups. click to reload'}</Text>
```

Comment:
we can do better than this

Reply:
Refreshed, found no easy answer, this works for POC